File: IMS/IMS.Presentation/DTOs/FeedbackDTO.cs
namespace IMS.Presentation.DTOs;

public class FeedbackDTO
{
    public required Guid TicketId { get; set; } 

    public required Guid SentById { get; set; } 

    public required Guid AddressedToId { get; set; } 

    public required string Comment { get; set; }
}

File: Frontend/ims-frontend/src/entities/ims/dto/feedback_dto.ts
export interface FeedbackDto {
    id: string;
    ticketId: string;
    sentById: string;
    addressedToId: string;
    comment?: string | null;
}

export interface CreateFeedbackDto {
    ticketId: string;
    sentById: string;
    addressedToId: string;
    comment?: string | null;
}

export interface UpdateFeedbackDto {
    comment?: string | null;
}
